### ISSUE TITLE
Tablet causing Mouse to stay on screen

### ISSUE THREAD

**Issue by MistJules** [civilnetworks/SCP_Public_Development#297]
When using the Personnel tablet and swapping off it causes the cursor to stay locked on the screen till you swap back to the tablet and deactivate it. Not even F3 will relock the mouse you can only stop it when you swap back to the tablet.

This also happens with the GOC drop tablet.

Clip: https://medal.tv/games/garrys-mod/clips/OpFWC7TFPALxZ/d1337Fa0XTeL?invite=cr-MSwzNjQsNzAxMDI3NjAs

**Comment by Kw1ll** [comment]
@MistJules can you add a type to these tickets please, helps a bit

**Comment by GarrySwepman** [comment]
Yeah I had this happen to me a few months ago, I had to rejoin it really sucked.

**Comment by KaptianCore** [comment]
For this bug u just need to exit out of the tablet again usually using your number keys on your keyboard to switch back to the tablet

**Comment by CNCanoon** [comment]
To be checked by content to see if this is still an issue.

**Comment by BlueCivilnetwork** [comment]
Still an Issue @CNCanoon .

**Comment by SynergyJohn** [comment]
Still an issue

**Comment by SynergyJohn** [comment]
Update, it seems to not only be when you have a tablet yourself, as it was found when I was on civilian without a tablet. After giving myself a tablet to attempt to fix it, it didn't help at all.
https://medal.tv/games/garrys-mod/clips/m6XFFsYYT5YWuqH7E?invite=cr-MSx6UG8sMTcwNTAxNDgw&v=35
```
Warning! Detected missing cam.End2D call!
Warning! Detected missing render.PopRenderTarget call!

[vwar] addons/vwar/lua/weapons/vwar_tactical_tablet/cl_init.lua:624: bad argument #1 to 'SetMaterial' (IMaterial expected, got nil)
  1. SetMaterial - [C]:-1
   2. RenderTablet - addons/vwar/lua/weapons/vwar_tactical_tablet/cl_init.lua:624
    3. unknown - addons/vwar/lua/weapons/vwar_tactical_tablet/cl_init.lua:670
     4. unknown - addons/cnlib/lua/includes/modules/hook.lua:422
```
